src/main/kotlin/apply/ui/api/MailRestController.kt

```diff
@@ -0,0 +1,29 @@
+package apply.ui.api
+
+import apply.application.mail.MailSendData
+import apply.application.mail.MailService
+import org.apache.commons.io.IOUtils
+import org.springframework.core.io.ByteArrayResource
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestPart
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.multipart.MultipartFile
+
+@RestController
+@RequestMapping("/api/mail")
+class MailRestController(
+    private val mailService: MailService
+) {
+    @PostMapping
+    fun sendMail(
+        @RequestPart(value = "request") request: MailSendData,
+        @RequestPart(value = "files") files: Array<MultipartFile>,
+    ): ResponseEntity<Unit> {
```

Comment:
```suggestion
    @PostMapping
    fun sendMail(
        @RequestPart request: MailSendData,
        @RequestPart files: Array<MultipartFile>,
    ): ResponseEntity<Unit> {
```
굳굳 !! 
요청 키값과 변수명이 같다면 value를 생략해도 좋을 것 같아요.

Reply:
확인했습니다. `RequestPart` 부분을 처음 써서 명시해줬는데 없어도 상관없다면 지워줘도 되겠네요